app/gui/src/ide.rs

```diff
@@ -74,13 +74,13 @@ impl Ide {
 
         enso_frp::extend! { network
             on_log_sent          <- source::<()>();
-            mouse_moved          <- mouse.position.constant(());
-            any_mouse_press      <- any(mouse.up,mouse.down).constant(());
-            any_mouse_event      <- any(any_mouse_press,mouse_moved,mouse.wheel);
-            any_keyboard_event   <- any(keyboard.down,keyboard.up).constant(());
-            any_input_event      <- any(any_mouse_event,any_keyboard_event);
+            mouse_moved          <- mouse.position.constant(()).profile();
+            any_mouse_press      <- any(mouse.up,mouse.down).constant(()).profile();
+            any_mouse_event      <- any(any_mouse_press,mouse_moved,mouse.wheel).profile();
+            any_keyboard_event   <- any(keyboard.down,keyboard.up).constant(()).profile();
+            any_input_event      <- any(any_mouse_event,any_keyboard_event).profile();
             // True if any input event was captured since the last "alive" log sending.
-            input_event_received <- bool(&on_log_sent,&any_input_event).sampler();
+            input_event_received <- bool(&on_log_sent,&any_input_event).profile().sampler();
         }
```

Comment:
Do we want to leave it in the source code? I was sure that `profile` FRP nodes should e used for debug purposes only, not in order to leave them forever there. Especially, as their names might be misleading (are the names auto-generated by the FRP macro? I think they are, otherwise these would be unnamed endpoints). So you have "mouse_moved" profile, which measures not only moved mouse but everything connected to this FRP endpoint, which can be a very heavyweight operation.